textpipe/pipeline.py

```diff
@@ -37,3 +38,37 @@ def __call__(self, raw):
         doc = Doc(raw, language=self.language, hint_language=self.hint_language)
         result_dict = dict(zip(self.pipeline, [doc.__getattribute__(x) for x in self.pipeline]))
         return result_dict
+
+    def save(self, filename):
+        """
+        Serializes the pipeline to file (using json)
+
+        Args:
+        filename: location of where to serialize pipeline
+
+        >>> filename = "test.json"
+        >>> Pipeline(['n_sents', 'clean_text']).save(filename)
+        >>> open(filename).read()
+        '{"pipeline": ["n_sents", "clean_text"], "language": null, "hint_language": null}'
+        """
+
+        with open(filename, 'w') as json_file:
+            json.dump(self.__dict__, json_file)
+
+    @staticmethod
+    def load(filename):
+        """
+        Loads pipeline from serialized file
+
+        Args:
+        filename: location of serialized Pipeline object
+
+        >>> filename = "test.json"
+        >>> Pipeline(['n_sents', 'clean_text']).save(filename)
+        >>> p = Pipeline.load(filename)
+        >>> p.__dict__
+        {'pipeline': ['n_sents', 'clean_text'], 'language': None, 'hint_language': None}
```

Comment:
I think dict equality doesn't really work. You can compare a sorted list of tuples instead

`sorted(p.__dict__.items())`